<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edycyjne</class>
 <widget class="QDialog" name="Edycyjne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>307</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Kolor czcionki</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="fontColor">
               <item>
                <property name="text">
                 <string>Czerwony</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Zielony</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Niebieski</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Styl czcionki</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="fontStyle">
               <property name="minimumSize">
                <size>
                 <width>210</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>210</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>Normalny</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Kursywa</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pogrubiony</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonIMG">
           <property name="text">
            <string>Wybierz zdjecie</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumTime">
            <time>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
           <property name="calendarPopup">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Edycyjne</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Edycyjne</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
